packages/app/client/src/ui/pages/Account/components/Profile.js

```diff
@@ -1,9 +1,242 @@
+    border: '1.2px solid #E7E7E7',
+    borderRadius: '5px',
+    padding: '2%',
+    marginBottom: '2%'
+  },
+  deleteButton: {
+    backgroundColor: 'white',
+    minWidth: 0,
+    width: '12px',
+    height: '13px'
+  },
+  fileInput: {
+    width: 0.1,
+    height: 0.1,
+    opacity: 0,
+    overflow: 'hidden',
+    position: 'absolute',
+    zIndex: -1
+  },
+  fileInputLabel: {
+    cursor: 'pointer'
+  },
+  addFileButton: {
+    marginRight: '5%'
+  },
+  avatarSection: {
+    display: 'flex',
+    alignItems: 'center'
+  },
+  clarification: {
+    color: '#919492'
+  }
+}))
 
 function Profile(props) {
-    return (
-        <div>profile</div>
-    )
+  const inputRef = React.useRef(null)
+  const classes = useStyles()
+  const history = useHistory()
+  const [editting, setEditting] = useState(false)
+  const [websites, setWebsites] = useState(props.websites)
+  const [avatar, setAvatar] = useState(props.avatar)
+
+  const { updateUser } = useAuthAsyncActions()
+
+  const submitHandler = async userDetails => {
+    const formData = new FormData()
+    if (avatar) {
+      formData.append('avatar', avatar)
+    }
+    formData.append('name', userDetails.name)
+    formData.append('bio', userDetails.bio)
+    formData.append('role', userDetails.role)
+    formData.append('organization', userDetails.organization)
+    formData.append('websites', websites)
+    await updateUser(props.id, formData)
+    setEditting(false)
+  }
+
+  const fields = [
+    { ...name, initialValue: props.name || '', disabled: !editting },
+    {
+      ...bio,
+      initialValue: props.bio || '',
+      rows: 4,
+      multiline: true,
+      disabled: !editting
+    },
+    { ...role, initialValue: props.role || '', disabled: !editting },
+    {
+      ...organization,
+      initialValue: props.organization || '',
+      disabled: !editting
+    },
+    {
+      ...website,
+      disabled: !editting,
+      endAdornment: editting ? (
+        <InputAdornment position="end">
+          <Button
+            className={classes.enterButton}
+            onClick={() =>
+              setWebsites([
+                ...websites,
+                document.getElementById('websiteInput').value
+              ])
+            }
+          >
+            Enter
+          </Button>
+        </InputAdornment>
+      ) : null
+    }
+  ]
+
+  return (
+    <MainContainer size="small" mt="-50px" margin="0">
+      <SimpleFormWithHeader
+        fields={fields}
+        title="Profile"
+        onSubmit={
+          editting
+            ? submitHandler
+            : (_, formProps) => {
+                setEditting(true)
+                formProps.setSubmitting(false)
+              }
+        }
+        submitLabel={editting ? 'Save changes' : 'Edit'}
+        titleProps={{ align: 'left' }}
+        buttonLayout={{ xs: 4 }}
+        validate={validateProfileForm}
+        renderBellowForm={
+          <>
+            <Grid item xs={12} className={classes.avatarSection}>
```

Comment:
You don’t have any component alongside the Grid wrapper so I think you don’t need the Fragments here